sched/Kconfig

```diff
@@ -219,7 +219,7 @@ config START_DAY
 
 config MAX_WDOGPARMS
 	int "Maximum number of watchdog parameters"
-	default 4
+	default 2
```

Comment:
I think if you change CONFIG_MAX_WDOGPARMS to 2, you should also change the prototype of wd_start() from:

  int wd_start(WDOG_ID wdog, int32_t delay, wdentry_t wdentry, int argc, ...);

to

  int wd_start(WDOG_ID wdog, int32_t delay, wdentry_t wdentry, wdparm_t arg1, wdparm_t arg2);

Problems with the use of the variadic wd_start() was also by @yamt and others in a previous PR.  At that time, they wanted to change the `int argc, ...` to a structure pointer.  I resisted that change but if _CONFIG_MAX_WDOGPARMS_ is set to 2, then there is no reason to pass a variable argument and we should then honor the concerns of @yamt and others.

Reply:
BTW.  I don't know how you are going to get this merged.  It is impossible to review or even to see all 534 files.

Reply:
If you add `.diff` to the PR url you will get the full diff.

Reply:
@patacongo so do you want to remove option MAX_WDOGPARMS and fix the argument to 2?

Reply:
I'm not certain.  If two is enough then this would also resolve other type-related concerns.  There is a risk in passing arbitrary parameters that may not be cast -compatible with wdparm_t.

I don't personally care very much.  Do you think there is a need for an arbitrary number of parameters?

Reply:
Let's get input from @yamt he did have an opinion in the past.

Reply:
Ok, from the technical point, after @YuuichiNakamura's patch: https://github.com/apache/incubator-nuttx/issues/740, the variable arguments should work as expect without problem.

Reply:
> Ok, from the technical point, after @YuuichiNakamura's patch: #740, the variable arguments should work as expect without problem.

It is in PR #740 where @yamt supported changing the wd_start() prototype so that it is no longer a variadic function.